src/main/java/reposense/util/FileUtil.java

```diff
@@ -38,6 +41,52 @@
     private static final String GITHUB_API_DATE_FORMAT = "yyyy-MM-dd";
     private static final ByteBuffer buffer = ByteBuffer.allocate(1 << 11); // 2KB
 
+    /**
+     * Returns a {@code set} of relevant repo paths from {@code repoConfigsList}.
+     */
+    public static Set<Path> getReportFolders(Path sourcePath, List<RepoConfiguration> repoConfigsList) {
+        Set<Path> relevantFolders = new HashSet<>();
+        for (RepoConfiguration repoConfiguration : repoConfigsList) {
+            relevantFolders.add(Paths.get(sourcePath + File.separator + repoConfiguration.getDisplayName())
+                    .toAbsolutePath());
```

Comment:
The readability of this line needs to improve

Either do this

```
            relevantFolders.add(
                    Paths.get(sourcePath + File.separator + repoConfiguration.getDisplayName()).toAbsolutePath());
```

or create a variable to concat and reuse the path
`sourcePath + File.separator + repoConfiguration.getDisplayName()`